lib/widgets/tb_elevated_button.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:tech_byte/utils/colors.dart';
+import 'package:tech_byte/utils/constants.dart';
+
+enum TBButtonType { primary, secondary }
+
+class TBButton extends StatelessWidget {
+  final void Function() onPressed;
+  final String text;
+  final bool enabled;
```

Comment:
There is no need for additional parameters.  Use onPressed, to determine if it is enabled or not, that is the core logic for Flutter built-in buttons why changing it?